xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminTranslations.xml

```diff
@@ -78,7 +78,7 @@ XWiki.XWikiPreferences_showattachments.hint=Display the attachments section in t
 XWiki.XWikiPreferences_showhistory.hint=Display the history section in the page layout.
 XWiki.XWikiPreferences_showinformation.hint=Display the information section in the page layout.
 XWiki.XWikiPreferences_title.hint=Define the page title format as seen by browsers.
-XWiki.XWikiPreferences_meta.hint=Define the meta data added to each page, specifying page description, author, keywords etc.
+XWiki.XWikiPreferences_meta.hint=Define the meta data added to each page, specifying page description, author, keywords etc. It is advised to use #template('meta.vm') as part of the content to benefit easily from future improvements of the default meta definition.
```

Comment:
isn't it `#template('meta')` without the .vm part? I have a doubt

Reply:
Based on examples from our codebase, seems like the `.vm` is required, for instance https://github.com/xwiki/xwiki-platform/blob/8bfe4069665fb16513159e8bd974c39b4f3094ff/xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml#L363

Reply:
@surli confirmed by testing with the code below in the meta field. The .vm part is indeed not optional.

```
<meta name="before" content="Home" />
#template('meta.vm')
<meta name="after" content="Home" />
```